core/src/rpc_pubsub.rs

```diff
@@ -895,8 +895,7 @@ mod tests {
         let (subscriber, _id_receiver, receiver) = Subscriber::new_test("voteNotification");
 
         // Setup Subscriptions
-        let subscriptions =
-            RpcSubscriptions::new(&exit, bank_forks.clone(), block_commitment_cache.clone());
+        let subscriptions = RpcSubscriptions::new(&exit, bank_forks, block_commitment_cache);
```

Comment:
@svenski123 Could you check the flakiness of this test `test_vote_subscribe` locally? it seems like this test got a lot unstable with removing `.clone()` on ci:

- https://buildkite.com/solana-labs/solana/builds/24611#4e149382-e000-4a94-85c3-c8f520e42009/467-993
- https://buildkite.com/solana-labs/solana/builds/24611#b6c44f63-29ad-41a5-a030-c16c22e8dc06/128-654
- https://buildkite.com/solana-labs/solana/builds/24611#b16ec14f-7317-477a-a337-a09bacd3a411/133-659
- https://buildkite.com/solana-labs/solana/builds/24611#b38fbc2e-0e10-485c-9133-9109113aedc7/116-642

My guess is that this or [that](https://github.com/solana-labs/solana/pull/10172/files#diff-9078d14bfb5de677e818bdca3c32a292R928) `.clone()` takes some time and removing that makes it more susceptible to the some kind of race condition. If you're brave enough, feel free to go down the rabbit hole or just add `#allow(...)`...

FYI, this test has been a bit flaky for some time, iirc.